src/backend/dx12/src/command.rs

```diff
@@ -1231,17 +1293,231 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
 
     fn blit_image<T>(
         &mut self,
-        _src: &n::Image,
+        src: &n::Image,
         _src_layout: image::Layout,
-        _dst: &n::Image,
+        dst: &n::Image,
         _dst_layout: image::Layout,
-        _filter: image::Filter,
-        _regions: T,
+        filter: image::Filter,
+        regions: T,
     ) where
         T: IntoIterator,
         T::Item: Borrow<com::ImageBlit>
     {
-        unimplemented!()
+        let device = self.shared.service_pipes.device.clone();
+
+        // TODO: Resource barriers for src and dst.
+        // TODO: depth or stencil images not supported so far
+
+        // TODO: only supporting 2D images
+        match (src.kind, dst.kind) {
+            (image::Kind::D2(..), image::Kind::D2(..)) => {},
+            _ => unimplemented!(),
+        }
+
+        // Descriptor heap for the current blit, only storing the src image
+        let srv_heap = {
```

Comment:
As I've just seen it I think we can use `Device::create_descriptor_heap_impl` here also.